stl/inc/chrono

```diff
@@ -5221,7 +5224,7 @@ namespace chrono {
 } // namespace chrono
 
 #if _HAS_CXX20

```

Comment:
Is that `_HAS_CXX20` subsumed by the `__cpp_lib_format`

Reply:
I didn't even check; I just replaced `__cpp_lib_concepts` with `__cpp_lib_format`. This was:
```c++
#if _HAS_CXX20
#ifdef __cpp_lib_concepts
[...]
#endif // __cpp_lib_concepts
#endif // _HAS_CXX20
```
before, so it was redundant then and now. I'll remove the `_HAS_CXX20` layer since `defined(__cpp_lib_format)` implies `_HAS_CXX20`.